content/ytt/docs/latest/lang-ref-string.md

```diff
@@ -127,3 +127,27 @@ Copied here for convenience from [Starlark specification](https://github.com/goo
 ```python
 "Hello, World!".upper()  # "HELLO, WORLD!"
 ```
+
+* [Multiline strings](https://github.com/google/starlark-go/blob/master/doc/spec.md#string-literals)
+  
+  Since ytt uses a modified version of Starlark that is not whitespace sensitive, to keep indentation in muli-line strings we recommend using a [`yamlfragment`](lang-ref-yaml-fragment.md).
```

Comment:
If I'm tracking what your intent is here in this commit, then _this_ is the place to provide some context for what's happening with multi-line strings.

Given that there's no better anchor into the Starlark spec than the one you included, the reader will likely appreciate a succinct introduction of what multi-line strings are (thinking a sentence or two)... I'd allow myself to be inspired by the text in the spec 😉 .

Then, a mention that `ytt`'s modification of Starlark does not preserve indentation (and I might refer the reader to the GitHub issue for details).

After that, this solution you give below can be described as a workaround the slightly diminished feature.

Reply:
Great feedback, I updated the description of multiline strings to be straight from the Starlark spec. Let me know if you have more thoughts